src/common/components/repository-row/dropdowns/remove-repo-dropdown.js

```diff
@@ -6,40 +6,102 @@ import { WarningIcon } from '../icons';
 
 import styles from './dropdowns.css';
 
+const getRemoveWarningMessage = (isBuilt, registeredName) => {
```

Comment:
Moved from `RepositoryRow` as it makes much more sense now to keep all remove-related logic in one place.